playground/terraform/README.md

```diff
@@ -16,74 +16,118 @@
-
-```bash
-touch /path/to/beam/playground/terraform/environment/{env-name}/terraform.tfvars
+project_id           = "project_id"          #GCP Project ID
+network_name         = "network_name"        #GCP VPC Network Name for Playground deployment
+gke_name             = "playground-backend"  #Playground GKE Cluster name
+region               = "us-east1"            #Set the deployment region
+location             = "us-east1-b"          #Select the deployment location from available in the specified region
+state_bucket         = "bucket_name"         #GCS bucket name for Beam Playground temp files, [terraform state file] (https://cloud.google.com/docs/terraform/resource-management/store-state)
+bucket_examples_name = "bucket_name-example" #GCS bucket name for Playground examples storage
 ```
-
-Then provide necessary variables.
-
-## 2. Provision infrastructure
-
-To deploy Playground infrastructure run gradle task:
-
-```bash
-./gradlew playground:terraform:InitInfrastructure -Pproject_environment="env-name"
+* `state.tfbackend` environment variables:
 ```
-
-## 3. Deploy application
-
-To deploy application run following steps:
-
-* Authinticate in Artifact registry
-
-```bash
-gcloud auth configure-docker us-central1-docker.pkg.dev
+bucket               = "bucket_name"         #input bucket name - will be used for terraform tfstate file
 ```
+Then, configure authentication for the Google Cloud Platform:
 
-* Вeploy backend services
-
-```bash
-./gradlew playground:terraform:deployBackend -Pproject_environment="env-name" -Pdocker-tag="tag"
+* The following commands allows us to configure account and project
+```
+gcloud init
+```
 ```
+gcloud auth application-default login
+```
+# Infrastructure deployment:
+* To deploy the Payground infrastructure, use the following command (please be sure that you are in the "beam" folder):
+```
+./gradlew playground:terraform:InitInfrastructure -Pproject_environment="env" (env - folder name which you created for configuration files)
+```
+# Playground deployment:
+Afrer successfully deploying Playground infrastrcuture in the previous step, authenticate on Docker and Google Kubernetes Engine<br>
+The following command will authenticate us in the Docker registry
+```
+ gcloud auth configure-docker `chosen_region`-docker.pkg.dev
+```
+The following command will authenticate in GKE
+```
+gcloud container clusters get-credentials --region `chosen_location` `gke_name` --project `project_id`
+```
+Find a Static IP in your GCP project>VPC Network>IP Addresses>pg-static-ip
+<br>Add following DNS A records for the Static IP address:
+```
+* java.playground.zone
+* python.playground.zone
+* scio.playground.zone
+* go.playground.zone
+* router.playground.zone
+* playground.zone
+```
+Where "playground.zone" is your registered DNS zone<br>
+[More about DNS zone registration](https://domains.google/get-started/domain-search/)<br>
+[More about A records in DNS](https://support.google.com/a/answer/2579934?hl=en)
 
```

Comment:
DNS names changes can take up to 72 hours to propagate worldwide.

Reply:
fixed